client/state/action-types.js

```diff
@@ -72,6 +72,9 @@ export const FOLLOWER_REMOVE_SUCCESS = 'FOLLOWER_REMOVE_SUCCESS';
 export const GOOGLE_APPS_USERS_FETCH = 'GOOGLE_APPS_USERS_FETCH';
 export const GOOGLE_APPS_USERS_FETCH_COMPLETED = 'GOOGLE_APPS_USERS_FETCH_COMPLETED';
 export const GOOGLE_APPS_USERS_FETCH_FAILED = 'GOOGLE_APPS_USERS_FETCH_FAILED';
+export const GRAVATAR_UPLOAD_START = 'GRAVATAR_UPLOAD_START';
+export const GRAVATAR_UPLOAD_SUCCESS = 'GRAVATAR_UPLOAD_SUCCESS';
+export const GRAVATAR_UPLOAD_FAILURE = 'GRAVATAR_UPLOAD_FAILURE';
```

Comment:
While we're not super consistent, I'd suggest the following:
```javascript
export const GRAVATAR_UPLOAD_RECEIVE = 'GRAVATAR_UPLOAD_RECEIVE';
export const GRAVATAR_UPLOAD_REQUEST = 'GRAVATAR_UPLOAD_REQUEST';
export const GRAVATAR_UPLOAD_REQUEST_SUCCESS = 'GRAVATAR_UPLOAD_REQUEST_SUCCESS';
export const GRAVATAR_UPLOAD_REQUEST_FAILURE = 'GRAVATAR_UPLOAD_REQUEST_FAILURE';
```

The last event is used to dispatch data that we'd store in our reducer. So typically after a successful request we'd trigger both `GRAVATAR_UPLOAD_REQUEST_SUCCESS` and `GRAVATAR_UPLOAD_RECEIVE`. It's nice to separate these two events though, since it makes it clearer as to what's happening in the reducer.